core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -497,6 +497,15 @@ paths:
           schema:
             type: string
             format: date
+        - name: aggregationKey
+          in: query
+          description: Key indicating how to aggregate spend data in results
+          required: false
```

Comment:
If the aggregation key is not present, will you only get the summary data?

Reply:
At the moment it will default to `Daily`, but I could change it to only return summary data

Reply:
I see my answer in the code below… Daily is the default value. If we want to stick with that, I think it should be documented here. Or I personally think it would be best to only return the summary data if an aggregationKey is not specified.